src/app/components/team/SlackConfig.js

```diff
@@ -166,7 +166,10 @@ class SlackConfig extends React.Component {
           onClose={this.handleCloseDialog}
         >
           <DialogTitle>
-            {this.props.intl.formatMessage(messages.title)}
+            <FormattedMessage 
+              id={messages.title.id} 
+              defaultMessage={messages.title.defaultMessage} 
```

Comment:
The desired usage is that the actual content of these keys is moved here, and `title` is to be removed from the `messages` object.